macvtap_test.go

```diff
@@ -29,7 +29,7 @@ import (
 )
 
 const MASTER_NAME = "eth0"
-const PrivateMACPrefixString = "0a:59"
+const PrivateMACPrefixString = "02:00:00"
```

Comment:
02 is enough for local unicast, isn't it

Reply:
Removed that const, we're not generating mac addresses, and as such, makes no sense trying to assure namespaces.